src/integrationTest/java/org/openstreetmap/atlas/geography/atlas/dynamic/DynamicAtlasIntegrationTest.java

```diff
@@ -31,35 +31,35 @@
     public void testAreas()
     {
         final Atlas atlas = new DynamicAtlas(POLICY);
-        Assert.assertEquals(31042, Iterables.size(atlas.areas()));
+        Assert.assertEquals(39672, Iterables.size(atlas.areas()));
```

Comment:
Why are these values changing?

Reply:
Good question! The two DMA atlas shards that are used in that test have been updated so they now have consistent edges at the borders, which was not the case before (and which was why we needed the MABF). @MikeGost who did that update can add more, if I am missing something?

Reply:
That's right.